plugins/ida/ida_main.ml

```diff
@@ -166,12 +252,70 @@ let checked ida_path is_headless =
        (ida_path/"plugins"/"plugin_loader_bap.py"))  >>| fun () ->
   ida_path
 
+let run_ko_symbol_mapper_pass ida_futures =
```

Comment:
This pass changes the Cfg recovered from disasm (BIL), which is then propagated to the IR construction. Therefore this pass must run after collecting branch information, but before other passes like API which manipulate the IR. For instance, if this were a standalone plugin, we would have to run passes like API again after running this pass, which seems redundant. The current solution is to run this pass right after the IDA information is available.